test/unit/sz.c

```diff
@@ -0,0 +1,55 @@
+#include "test/jemalloc_test.h"
+
+static size_t
+reg_size_compute(int lg_base, int lg_delta, int ndelta) {
+    return (ZU(1) << lg_base) + (ZU(ndelta) << lg_delta);
+}
+
+TEST_BEGIN(test_sz_psz2ind) {
+    pszind_t zero_ind = sz_psz2ind(0);
+    expect_lu_eq(zero_ind, zero_ind, "sz_psz2ind(0) can't run");
```

Comment:
This test will never fail, why is it needed? A better way to set argument requirements is via `assert` in the function itself.